<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>582</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>NPM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NAMA LENGKAP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NAMA PANGGILAN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TELPON</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>EMAIL</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>KELAS</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MATA KULIAH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LOKASI KAMPUS</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_NIM">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nama">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_panggilan">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_telpon">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_kelas">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_kuliah">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lokasi">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_ubah">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>UBAH</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>download (3).png</normaloff>download (3).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_hapus">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HAPUS</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>download (1).png</normaloff>download (1).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_batal">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>BATAL</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>download (2).png</normaloff>download (2).png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableMahasiswa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>851</width>
      <height>231</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>NPM</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NAMA LENGKAP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>NAMA PANGGILAN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>TELPON</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EMAIL</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>KELAS</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MATA KULIAH</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>LOKASI KAMPUS</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string>                                                         MAHASISWA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_tambah">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>TAMBAH</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>download.png</normaloff>download.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
